### PR TITLE
Horizontal lines overlapping bars

### PR DESCRIPTION
**Pull request by AlexanderK79**
Following https://github.com/microsoft/powerbi-visuals-gantt/issues/153 I made an adjustement that resolves the described issue.

Also while scaling the Task height

### CODE REVIEW

File: src/gantt.ts [hunk 1]
```diff
@@ -1902,7 +1903,8 @@ export class Gantt implements IVisual {
                     const isLastChild = childrenCount && childrenCount === currentChildrenIndex;
                     return drawStandartMargin || isLastChild ? Gantt.DefaultValues.ParentTaskLeftMargin : Gantt.DefaultValues.ChildTaskLeftMargin;
                 })
-                .attr("y", (task: GroupedTask) => Gantt.DefaultValues.TaskLineWidth + (task.id + 1) * this.getResourceLabelTopMargin())
+                //.attr("y", (task: GroupedTask) => Gantt.DefaultValues.TaskLineWidth + (task.id + 1) * this.getResourceLabelTopMargin())
```
Comment (zBritva) [inline src/gantt.ts]: Please, remove the commented code.

Comment (AlexanderK79) [reply src/gantt.ts]: Sorry, first time working on GitHub.... :-)

**Comment by zBritva** [comment]
@AlexanderK79 Could you please update version number of the visual in pbiviz.json and package.json; And also add short description of fix in changelog.md.
Test are passed, we can merge your changes.
Thanks!

**Comment by AlexanderK79** [comment]
I have updated and committed the requested files.... hope I did it right :-)

**Comment by zBritva** [comment]
> I have updated and committed the requested files.... hope I did it right :-)

Yes, but you forget displayName: "Gantt 2.0.1". You need to change to 2.0.2 also. 
PBI displays text as name in UI for users.
Thanks!

**Comment by zBritva** [comment]
@AlexanderK79 don't add commits to master, before merge this PR.

**Comment by AlexanderK79** [comment]
OK
sorry!
will pay attention from now on

**Comment by th340385** [comment]
Very new to this; however, will this update be automatically pushed through to Power BI? How long does that take?

**Pull request closed without merging**